dockerfiles/cli/cli-binary-dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM harbor.galasa.dev/docker_proxy_cache/library/httpd:2.4.59
+FROM harbor.galasa.dev/docker_proxy_cache/library/httpd:alpine
```

Comment:
We actually can delete this dockerfile altogether as the one we use lives in CLI (I forgot to clean this up!)

Reply:
That's great, thanks for letting me know! Removed in https://github.com/galasa-dev/automation/pull/631/commits/b8de69c392724e323984b6684c300c08a5124b43

Reply:
As can the other two dockerfiles in the same directory